src/__tests__/upload.js

```diff
@@ -163,3 +163,55 @@ test('should call onChange/input bubbling up the event when a file is selected',
   expect(onInputInput).toHaveBeenCalledTimes(1)
   expect(onInputForm).toHaveBeenCalledTimes(1)
 })
+
+test('should upload file with accepted format', () => {
```

Comment:
Minor style issue: The tests in this repo follow the `imperative` style. So just `upload` instead of `should upload`.

Reply:
But the other test cases that were already this file are using `should` 🤔

Reply:
It's not just in this file btw, there are a lot of other tests mixing imperative style with declarative style - Should we fix that in a separate PR?

Reply:
Just checked it: Actually nothing mentioned in the guideline. Guess it's just that everybody used that `imperative` style around here recently. I guess everybody just brings their style and each suite reflects who contributed most/first to it.

Reply:
I've actually have sent a PR to fix that but I messed up and created a branch with the changes of this PR 🤕 

I'll submit another one with isolated changes and we can discuss it there